docs/azure_service_principal.md

```diff
@@ -0,0 +1,77 @@
+## Create a Service Principal Account
+Pick any one of the two different ways to create a Service Principal.
+ - Azure Portal
+ - Azure CLI
+ 
+## Using Azure Portal:
+ - Login to your Azure account on https://portal.azure.com
+ - Navigate to `User Settings` under `Azure Active Directory`.
+ - Ensure that user can register applications, ONLY an admin user can register applications.
+ - Create an `Azure Active Directory application` [(Learn more)](https://docs.microsoft.com/en-us/azure/azure-resource-manager/resource-group-create-service-principal-portal?view=azure-cli-latest).
+ - Retrieve `application ID` and `authentication key`
+ - Assign the newly created application to a role in order to access Azure resources.
```

Comment:
I think it is better to say here: Assign the contributor role to the created service principal, over all storage accounts that are needed to get accessed by the application.

Reply:
ok